### ISSUE TITLE
Error logging - moving away from email

### ISSUE THREAD

**Issue by ArneTR** [green-coding-solutions/green-metrics-tool#473]
Since we already have a file logging implemented in the https://github.com/green-coding-berlin/green-metrics-tool/blob/main/lib/error_helpers.py we should:

- Define a file where to log runner errors
- Define a file where to log api errors (My suggestion would be to bind a writeable file into the container as bind-mount)
- The user then can define a transport mechanism of choice, for instance pushing all logs into a logstash etc.

Currently every failure in the API takes a lot of processing time. So even pushing to a remote service would create quite some load.

The filesystem solution decouples it completely.

Our implementation to have then again email, but async, could be using [swatch](https://github.com/green-coding-berlin/green-metrics-tool/blob/main/lib/error_helpers.py)

**Comment by ribalba** [comment]
What about we implement https://docs.python.org/3/library/logging.html . This is something I have had on my todo for some time. We could implement logging for the runner and the server and such have a unified api. We could also start the server/ runner in debug mode to see what is happening, if we need to and otherwise reduce the "clutter" a little.

**Comment by ArneTR** [comment]
yes, can be done. We can just swap it for the standard library to make it more semantic / obvious for frequent Python users.

The logging is already centralized and unified in server and CLI though, so from a functional perspective I see no uptake ... but still agree to do it once we tackle it.

**Comment by ArneTR** [comment]
Issue has been handled months ago.

All errors are saved to the database and can be picked up there by an arbitrary handler. By default this is email.

If that is not needed GMT can also only log to the filesytem
